packages/openactive-integration-tests/test/flows/booking/book.js

```diff
@@ -1,5 +1,6 @@
 const chakram = require("chakram");
 const RequestHelper = require("../../helpers/request-helper");
+const validationHelper = require("../../helpers/validation-helper");
```

Comment:
maybe call directory shared-tests to make it clearer it defines tests rather than being a standalone little helper function

Reply:
then file could just be validation and the const could be sharedValidationTests (or something)

Reply:
fine with it being called just /shared?